src/admin/python/lsst/qserv/admin/itest_load.py

```diff
@@ -28,12 +28,12 @@
 import json
 import logging
 import mysql.connector
-from mysql.connector.cursor import MySQLCursor
+from mysql.connector.abstracts import MySQLCursorAbstract
```

Comment:
Also `MySQLConnectionAbstract` to shorten some of the lines below?

Reply:
Yes, thanks!